packages/graphql/src/schema/validation/directives.ts

```diff
@@ -163,6 +168,9 @@ export const relationshipDirective = new GraphQLDirective({
         type: {
             type: new GraphQLNonNull(GraphQLString),
         },
+        queryDirection: {
+            type: RelationshipQueryDirectionEnum,
+        },
```

Comment:
**suggestion (blocking):** This should have a default value of `DEFAULT_DIRECTED` should it not? I know we don't use these directive definitions explicitly, but they should mirror our expectations of the directive.